packages/table/src/Cell/Cell.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { Cell, CellProps } from '.';
+
+const onScroll = jest.fn();
+
+const defaultProps: CellProps = {
+  onScroll,
+};
+
+function renderCell(props: CellProps) {
+  return render(
+    <table>
+      <tbody>
+        <tr>
+          <Cell {...props} data-testid="lg-test-cell" />
+        </tr>
+      </tbody>
+    </table>,
+  );
+}
+
+describe('packages/table/Cell', () => {
+  describe('a11y', () => {
+    test('does not have basic accessibility issues', async () => {
```

Comment:
should we test that it renders its children?

Reply:
No need to at the Cell level - Cell just returns `children` without any processing. All rendering overrides are done at the Row level or above.